public_dropin_gpu_environments/triton_server/dr_requirements.in

```diff
@@ -1,4 +1,5 @@
 requests
+openai
```

Comment:
I'll move the base class out of __init__; that should fix it right? Leaving it there will just lead to more bugs where any other GPU predictor needs to depend on that library for no reason.